packages/accordion/README.md

```diff
@@ -0,0 +1,273 @@
+[//]: # 'AUTO INSERT HEADER PREPUBLISH'
+
+# Accordion
+
+`lion-accordion` is a component used to toggle the display of sections of content.
+Its purpose is to reduce the need to scroll when presenting multiple sections of content on a single page. Accordions often allow users to get the big picture before focusing on details.
+
+```js script
+import { LitElement } from 'lit-element';
+import { html } from 'lit-html';
+import './lion-accordion.js';
+
+export default {
+  title: 'Others/Accordion',
+};
+```
+
+```js preview-story
+export const main = () => html`
+  <lion-accordion>
+    <h3 slot="invoker">
+      <button>Lorem</button>
+    </h3>
+    <p slot="content">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+    </p>
+    <h3 slot="invoker">
+      <button>Laboriosam</button>
+    </h3>
+    <p slot="content">
+      Laboriosam sequi odit cumque, enim aut assumenda itaque quis voluptas est quos fugiat unde
+      labore reiciendis saepe, iure, optio officiis obcaecati quibusdam.
+    </p>
+  </lion-accordion>
+`;
+```
+
+## How to use
+
+### Installation
+
+```bash
+npm i --save @lion/accordion;
```

Comment:
No need for `;` (i see its copied of tabs)

Reply:
Done!